src/context/WalletProvider/NativeWallet/components/NativeImport.tsx

```diff
@@ -60,7 +61,7 @@ export const NativeImport = ({ history }: RouteComponentProps) => {
                 },
                 validate: {
                   validMnemonic: value =>
-                    bip39.validateMnemonic(value) ||
+                    bip39.validateMnemonic(value.trim()) ||
```

Comment:
It will only be validated in trimmed form, not displayed in trimmed from.
I'd expect either the seed to be left original for the user (only transformed under the hood for validation), or both transformations to be applied visually - not one or the other.

Reply:
Indeed, +1.